modules/installation-guide/partials/proc_using-the-operator-to-configure-communication-between-che-components.adoc

```diff
@@ -0,0 +1,30 @@
+[id="using-the-operator-to-configure-communication-between-{prod-id-short}-components_{context}"]
+= Using the Operator to configure communication between {prod-short} components
+
+To change between external and internal inter-component communication, update the Custom Resource (CR).
+
+.Prerequisites
+
+* An instance of {prod-short} running on {platforms-name}.
+* You have installed {prod-short} by using the Operator.
+
+.Procedure
+
+* Configure the `__<property-value>__` value of the `disableInternalClusterSVCNames` property in the CheCluster Custom Resource server settings. Set the value of `__<property-value>__` to:
+[horizontal]
+`true`:: To use external {platforms-ingress}.
+`false`:: To use internal {orch-name} DNS names.
+
++
+====
+[source,yaml,subs="+quotes"]
+----
+apiVersion: org.eclipse.che/v1
+kind: CheCluster
+# ...
+spec:
+  server:
+    # ...
+    disableInternalClusterSVCNames: __<property-value>__
+----
+====
```

Comment:
Verifications steps:

* Verify the value of following fields in the `che` ConfigMap: 
  CHE_API_INTERNAL
  CHE_WEBSOCKET_ENDPOINT__MINOR
  CHE_WEBSOCKET_ENDPOINT
  CHE_WORKSPACE_DEVFILE__REGISTRY__INTERNAL__URL
  CHE_KEYCLOAK_AUTH__INTERNAL__SERVER__URL
  CHE_WORKSPACE_PLUGIN__REGISTRY__INTERNAL__URL



When using internal services each FQDN ends with .svc
When using internal services each FQDN does not end with .svc

Example using internal services
====
  CHE_API_INTERNAL: 'http://che-host.openshift-workspaces.svc:8080/api'
  CHE_WEBSOCKET_ENDPOINT__MINOR: 'ws://che-host.openshift-workspaces.svc:8080/api/websocket-minor'
  CHE_WEBSOCKET_ENDPOINT: 'ws://che-host.openshift-workspaces.svc:8080/api/websocket'
  CHE_WORKSPACE_DEVFILE__REGISTRY__INTERNAL__URL: 'http://devfile-registry.openshift-workspaces.svc:8080'
  CHE_KEYCLOAK_AUTH__INTERNAL__SERVER__URL: 'http://keycloak.openshift-workspaces.svc:8080/auth'
  CHE_WORKSPACE_PLUGIN__REGISTRY__INTERNAL__URL: 'http://plugin-registry.openshift-workspaces.svc:8080/v3'
====

Reply:
I took a slightly different approach to make it easier to follow, but the key steps remain the same.